python/src/lib.rs

```diff
@@ -101,11 +101,11 @@ struct Indexer {
 }
 
 #[pymethods]
-impl Indexer {
+impl InvertedIndexer {
```

Comment:
Can we get rid of `InvertedIndexer`?

Reply:
resolving based on anticipated features that could benefit from a pre-inverted index on the horizon